src/System.Device.Gpio/System/Device/Gpio/GpioController.cs

```diff
@@ -305,7 +305,7 @@ public void UnregisterCallbackForPinValueChangedEvent(int pinNumber, PinChangeEv
             int logicalPinNumber = GetLogicalPinNumber(pinNumber);
             if (!_openPins.Contains(logicalPinNumber))
             {
-                throw new InvalidOperationException("Can not remove callback for a pin that is not open.");
+                throw new InvalidOperationException($"Can not remove callback for pin {logicalPinNumber} because is not open.");
```

Comment:
```suggestion
                throw new InvalidOperationException($"Can not remove callback for pin {logicalPinNumber} because it is not open.");
```